doc/source/whatsnew/v2.0.0.rst

```diff
@@ -341,6 +341,33 @@ Now, the axes return an empty :class:`RangeIndex`.
    pd.Series().index
    pd.DataFrame().axes
 
+.. _whatsnew_200.api_breaking.to_latex:
+
+DataFrame to LaTeX has a new render engine
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The existing :meth:`DataFrame.to_latex` has been restructured to utilise the
+dual implementation previously available under :meth:`.Styler.to_latex`.
+The arguments signature is similar, albeit ``col_space`` has been removed since
+it is ignored by LaTeX engines. This also requires ``jinja2`` as a dependency
+which needs to be installed.
+The pandas options below are no longer used. The dual, existing, options that
```

Comment:
Since this is a breaking API change, does it make sense to remove those `display` options too?

Reply:
yes it does. But this PR was large enough that I thought I would try and break it into chunks. Removing the latex options and the LatexFormatter code was intended as an add on.

Reply:
Okay sounds good as a follow up. If/when they are removed it would be good to update this to say `The pandas options below are no longer used and have been removed`